.ci.yaml

```diff
@@ -369,19 +369,36 @@ targets:
       - bin/
       - .ci.yaml
 
+  - name: Linux framework_tests_slow
+    bringup: true
+    recipe: flutter/flutter_drone
+    timeout: 60
+    properties:
+      dependencies: >-
+        [ ]
+      shard: framework_tests
+      subshard: slow
+      tags: >
+        ["framework", "hostonly", "shard", "linux"]
+    runIf:
+      - dev/
+      - packages/flutter/
+      - packages/flutter_driver/
+      - packages/integration_test/
+      - packages/flutter_localizations/
+      - packages/fuchsia_remote_debug_protocol/
+      - packages/flutter_test/
+      - packages/flutter_goldens/
+      - packages/flutter_tools/
+      - bin/
+      - .ci.yaml
+
   - name: Linux framework_tests_misc
     recipe: flutter/flutter_drone
     timeout: 60
     properties:
       dependencies: >-
-        [
-          {"dependency": "goldctl", "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"},
-          {"dependency": "clang", "version": "git_revision:5d5aba78dbbee75508f01bcaa69aedb2ab79065a"},
-          {"dependency": "cmake", "version": "version:3.16.1"},
-          {"dependency": "ninja", "version": "version:1.9.0"},
-          {"dependency": "open_jdk", "version": "version:11"},
-          {"dependency": "android_sdk", "version": "version:33v6"}
-        ]
+        [ ]
```

Comment:
Remove these lines if `dependencies` is empty.
Can you remove the dependencies from `Windows framework_tests_misc` and `Mac framework_tests_misc`?

Reply:
I will before merging. Right now I've left it there so I can easily fill it back in. And I'll check on the windows and mac versions as well before merging.

Reply:
@jmagman

Do you happen to know if the machines are guaranteed to start from clean? I'm hesitant to believe that splitting these tests up got rid of all of the dependencies for `misc` and that `slow` only needs the "android_sdk" dependency.

Reply:
Hm, if they are missing a dependency they should fail, even if the dependency was previously installed on the bot.  I guess you saw that with android_sdk at some point.  `bringup: true` means if it fails it won't close the tree, you can see if it needs anything in post-submit.  New builders are _such_ a pain to test in PRs, as you've now learned. 🙂

Reply:
And if you need a follow-up PR to change the dependencies you _will_ be able to remove `bringup:true` in the PR since it won't be a brand new builder anymore.